src/constants.js

```diff
@@ -0,0 +1,36 @@
+const contractAddress = "0xBDe77Edb54D9672e56D338D0Bce90ECe640BDF26";
```

Comment:
Please remove this contract Address it is private id

Reply:
Updated